src/lib/content.js

```diff
@@ -50,18 +37,18 @@ function slugify(text) {
         .toLowerCase()              // Convert the string to lowercase letters
         .trim()                     // Remove whitespace from both sides of a string (optional)
         .replace(/\s+/g, '-')       // Replace spaces with hyphen
-		.replace(/[^\w-]+/g, '')   // Remove all non-word chars
-		.replace(/--+/g, '-')     // Replace multiple hyphen with single hyphen
-		.replace(/(^-|-$)/g, ''); // Remove leading or trailing hyphen
+        .replace(/[^\w-]+/g, '')   // Remove all non-word chars
+        .replace(/--+/g, '-')     // Replace multiple hyphen with single hyphen
+        .replace(/(^-|-$)/g, ''); // Remove leading or trailing hyphen
 }
```

Comment:
a lot of PR noise from this reformatting - recommend not doing this in bigger projects!